edudoc/delegate-handbook/delegate-handbook.md

```diff
@@ -964,29 +915,30 @@ If you decide to modify the details of a competition, including the advancement
+Other more dramatic options include increasing the number of competitors who advance to a subsequent round. You may also add an additional round entirely. The number of advancing competitors between rounds must still follow the maximum size as stated by [Regulation 9p1](https://www.worldcubeassociation.org/regulations/#9p1). In both cases, keep in mind that some competitors may have already left the venue if they were not expecting to advance to a subsequent round. Unless you have a good reason to expect that the majority of the competitors who could take advantage of modified advancement conditions will still be present (e.g. a large 2nd round of a popular event like 3x3x3 still ahead of you in the schedule), generally avoid changing advancement conditions.
 
-Per point 7.4 of the WCRP announcement criteria, you may not add an event to an ongoing WCA competition, except as an unofficial event.
+As per point 7.4 of the WCRP announcement criteria, you may not add an event to an ongoing WCA competition, except as an unofficial event, for which you would have to use a third-party service.
 
-Finally, remember that running ahead of schedule can cause some competitors to be "late" to their assigned groups, particularly if they are still referencing the original competition schedule. Do not disqualify competitors for being present at the originally scheduled time. If the same event is still ongoing, it is usually fairly simple to add them to an ongoing or future group. Even if this causes a small delay, being ahead of schedule means that you have the buffer time to accommodate it. However, competitors are ultimately responsible for appearing when their group is called, even if the competition is running ahead of schedule. If you feel that accommodating a late competitor is infeasible or otherwise detrimental to the rest of the competition, you are not required to allow them to compete.
+Finally, remember that running ahead of schedule can cause some competitors to come to their assigned groups late (according to the updated schedule) due to them being unaware of the change. Do not disqualify competitors for being present at the originally scheduled time. If the same event is still ongoing, it is usually fairly simple to add them to an ongoing or upcoming group. Even if this causes a small delay, being ahead of schedule means that you have the buffer time to accommodate the competitor. However, competitors are ultimately responsible for appearing when their group is called, even if the competition is running ahead of schedule. If you feel that accommodating a late competitor is infeasible or otherwise detrimental to the rest of the competition, you are not required to allow them to compete.
```

Comment:
I personally find "Finally, remember that running ahead of schedule can cause some competitors to come to their assigned groups late (according to the updated schedule) due to them being unaware of the change." very awkwardly worded and I personally think that the initial wording of that sentence is better.

Reply:
How about "arrive late to their assigned groups"?

Reply:
I'd make sure that it still includes the reasoning that they are possibly referencing an old version of the schedule.  I've seen many delegates/organizers not consider that when making their decisions, so I think it's a good callout.

Reply:
What about now?

Reply:
Looks good!